src/app/Http/Controllers/Operations/FetchOperation.php

```diff
@@ -22,7 +22,7 @@ protected function setupFetchOperationRoutes($segment, $routeName, $controller)
 
         if (count($matches[1])) {
             foreach ($matches[1] as $methodName) {
-                Route::post($segment.'/fetch/'.Str::kebab($methodName), [
+                Route::post($segment.'/fetch/'.Str::kebab($methodName).'/{single?}', [
```

Comment:
I'd name this parameter ```key``` rather than ```single```:
- ```single```- implies a boolean to me
- ```key``` - implies an ID to me